### DISCUSSION TITLE
Bepinex not working

### DISCUSSION THREAD

**Discussion by VinitBelam** [BepInEx/BepInEx#259] [Help and Q&A]
So i have playthrough with few mods. I downloaded bepinex and the mods and everything was working alright.

Then i booted my computer. After that i downloaded Valheim and bepinex again. Then i realized that in bepinex there was supposed to be "config, core, patchers and plugins" folder in it, but there was only core. Then i downloaded bepinex form different site. there was all the folders. then i moved the files and the folders in it to Valheim folder, like you are supposed to.

When it's done, launching the game normally opens console, and in main menu, there is supposed to be text in top left corner that shows how many plugins are installed. And of course, there was no console on launch and no text in top left corner in main menu. i also tried the other bepinex, that only has the core folder in it and didn't work either.
 
I would really like to continue my playthrough with the mods.

**Comment by ManlyMarco** [comment]
Go back to your original setup and try running the game as administrator. If that helps then you need to fix file permissions for the bepinex folder, possibly for the whole game folder.
